force-app/main/default/objects/External_Committee__c/validationRules/ESA_Restrict_Committee_Type.validationRule-meta.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ValidationRule xmlns="http://soap.sforce.com/2006/04/metadata">
+    <fullName>ESA_Restrict_Committee_Type</fullName>
+    <active>true</active>
+    <errorConditionFormula>ISCHANGED( External_Committee_Type__c )</errorConditionFormula>
```

Comment:
@deepakmulamalla Will External_Committee_Type__c always have value? What if the picklist is not selected? Are we not checking for ISNULL in addition to ISCHANGED? Was this explicitly a requirement?

Reply:
External_Committee_Type__c should never be changed once it is cloned and it is always a required field.